crates/conductor_api/src/conductor_api.rs

```diff
@@ -53,8 +54,10 @@ impl ConductorApi {
             CryptoMethod::Encrypt => (String::from("encrypt"), String::from("message")),
             CryptoMethod::Decrypt => (String::from("decrypt"), String::from("message")),
         };
-
-        send_json_rpc(self.0.clone(), payload, request_response)
+        // all crypto payloads are base64 encoded as we need to support arbitrary data and JSON
+        // handling is painful without some kind of encoding
+        let encoded_payload = base64::encode(&payload);
+        send_json_rpc(self.0.clone(), encoded_payload, request_response)
```

Comment:
This will *not* handle arbitrary binary data; it will only handle the *subset* of possible binary data that also corresponds to a valid UTF-8 `String`.

Reply:
@pjkundert you're correct, my comment is wrong